### PR TITLE
Phase fshift correction

### PR DESCRIPTION
**Pull request by talonmyburgh**
Updates:
- Option to apply phase corrections to tuning 1
- Neatening of delay_tracking interpolation loop

### CODE REVIEW

**Review by jack-h — APPROVED** [review]
At first glance I think this looks plausible. Would suggest running with some prints/logs to sanity check (e.g. presumably the corrections should be several orders of magnitude smaller than the thing they are correcting) prior to merging.

File: software/control_sw/src/cosmic_fengine.py [hunk 1]
```diff
@@ -892,6 +894,9 @@ def set_delays(self, delay_to_load, delay_rate_to_load, phase_to_load, phase_rat
             the rate of change of phase, in radians per second. This is the incremental phase
             which should be added to the current phase each second.
         :type phase_rate_to_load: ndarray{float}
+        :param phase_correction_factor: 4-long numpy array of factors which when multiplied by the coarse delay calculated
+            herein will yeild the second phase correction factor to be subtracted from phase_to_load. Units of this correction are in Hz.
```
[the remark this answers is not in the document]
Comment (talonmyburgh) [reply software/control_sw/src/cosmic_fengine.py]: Fixed - 'i' before 'e' except after 'c' clearly didn't stick during my education :/

**Review by jack-h — APPROVED** [review]

**Comment by talonmyburgh** [comment]
I've just checked. All looks pretty reasonable. 
For `ea14` the correction is actually one order of magnitude smaller than I expected... but I'll see how this correction plays out for other antenna.

**Comment by talonmyburgh** [comment]
For `ea14` the phase correction is actually an order of magnitude smaller than I expected, but going over the units, nothing seems wrong.
Will inspect on other antnenna.

**Pull request merged**